app/react/src/client/preview/index.js

```diff
@@ -13,11 +13,12 @@ import { selectStory } from './actions';
 import reducer from './reducer';
 
 // check whether we're running on node/browser
-const { navigator } = global;
 const isBrowser =
   navigator &&
+  navigator.userAgent &&
   navigator.userAgent !== 'storyshots' &&
-  !(navigator.userAgent.indexOf('Node.js') > -1);
+  !(navigator.userAgent.indexOf('Node.js') > -1) &&
+  !(navigator.userAgent.indexOf('jsdom') > -1);
```

Comment:
This is the main difference with Jest22, it depends on a higher version of jsdom, and is has a new userAgent